python/jax/carfac.py

```diff
@@ -2483,10 +2493,12 @@ def run_segment_jit(
     naps: neural activity pattern
     naps_fibers: neural activity of the different fiber types
         (only populated with non-zeros when ihc_style equals "two_cap_with_syn")
+    receptor_pot: receptor potential of ihc
     state: the updated state of the CARFAC model.
     BM: The basilar membrane motion
     seg_ohc & seg_agc are optional extra outputs useful for seeing what the
```

Comment:
Not your change: but lets split this comment to really have one per arg.

Reply:
Thanks Rob, do you mean the seg_ohc + seg_agc?